testing/bdd/features/deduplicator/1Sty1Reg4DupAtts.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
+<tt:tt ebuttp:sequenceIdentifier="ReSequenced1" ebuttp:sequenceNumber="12" ttp:cellResolution="32 15" ttp:clockMode="local" ttp:timeBase="clock" xml:lang="en-GB" xmlns:ebuttp="urn:ebu:tt:parameters" xmlns:ebutts="urn:ebu:tt:style" xmlns:tt="http://www.w3.org/ns/ttml" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:xml="http://www.w3.org/XML/1998/namespace">
+  <tt:head>
+    <tt:styling>
+      <!-- Should be replaced with one attribute -->
+      <tt:style xml:id="SEQ1.defaultStyle1" tts:fontFamily="sansSerif " /> 
+    </tt:styling>
+    <tt:layout>
+      <tt:region xml:id="SEQ1.bottomRegion" tts:origin="14% 60%" tts:extent="71.25% 24%" style="SEQ1.defaultStyle1 SEQ1.defaultStyle1 SEQ1.defaultStyle1 SEQ1.defaultStyle1"/>
+    </tt:layout>
+  </tt:head>
+  <tt:body>
+    <tt:div region="SEQ58.bottomRegion">
```

Comment:
The test may not pass, because the deduplicator would likely throw an error about `SEQ58.bottomRegion` not being found